### PR TITLE
fromJsonValidationError did not expand varargs, fixes #8560

### LINKED ISSUE #8560: fromJsonValidationError does not expand varargs
`ValidationError.fromJsonValidationError` does not expand it's varargs, causing multiple arguments to be passed as a single collection to the `ValidationError` constructor.  This is leading to unexpected error message formatting, like "The minimum length is WrappedArray(3)"

As of Play 2.6.17 current implementation is:

```scala
  def fromJsonValidationError(jve: JsonValidationError): ValidationError = {
    ValidationError(jve.message, jve.args)
  }
```

But it should be

```scala
  def fromJsonValidationError(jve: JsonValidationError): ValidationError = {
    ValidationError(jve.message, jve.args:_*)
  }
```

### Play Version (2.5.x / etc)

Play 2.6.16
Scala

### Expected Behavior

```scala
val jsonError = JsonValidationError("Testing, testing {1} {2} {3}", "one", "two", "three")
val validationError = fromJsonValidationError(jsonError)
jsonError.args(2) == "three"
validationError.args(2) == "three"
```

### Actual Behavior

```scala
// Throws java.lang.ArrayIndexOutOfBoundsException: 2
validationError.args(2) == "three" 
```

### PR DESCRIPTION
**Pull request by SheldonSoftdev**
Change fromJsonValidationError to use correctly expanding varargs

## Fixes

Fixes #8560

### CODE REVIEW

**Comment by lightbend-cla-validator** [comment]
Hi @softdevca,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

**Comment by SheldonSoftdev** [comment]
CLA signed

File: framework/src/play/src/test/scala/play/api/data/validation/ValidationSpec.scala [hunk 1]
```diff
@@ -238,4 +240,12 @@ class ValidationSpec extends Specification {
     }
   }
 
+  "ValidationError" should {
+    "Preserve varargs when converting a JsonValidationError to a Play ValidationError" in {
+      val jsonError = JsonValidationError("Testing, testing {1} {2} {3}", "one", "two", "three")
+      val validationError = ValidationError.fromJsonValidationError(jsonError)
```
Comment (rakeshhny) [inline framework/src/play/src/test/scala/play/api/data/validation/ValidationSpec.scala]: This class `ValidationError` is not shown in the imports where as I can see that `JsonValidationError` is shown in the imprts.

Test looks good.

Comment (SheldonSoftdev) [reply framework/src/play/src/test/scala/play/api/data/validation/ValidationSpec.scala]: Thank you for the review.

`ValidationError` is in the `play.api.data.validation` package, the same package as the test and thus doesn't need to be imported.

**Comment by SheldonSoftdev** [comment]
Closing pull request, will create another with the correct authorship and cleaner history.  Thank you.

**Pull request closed without merging**